klasses/migrations/0001_initial.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-03-24 19:06
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Bootcamp',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.TextField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Klass',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('klass_id', models.IntegerField()),
+                ('price', models.DecimalField(decimal_places=2, max_digits=20)),
+                ('recommended_installment', models.DecimalField(decimal_places=2, max_digits=20)),
```

Comment:
What is this for ?   I think they want each class to have a somewhat customizable nr of installments.  
```
class_1
     Installment_1 
          deadline:  03/15/2017
          min_amout:  $2000
     Installment_2 
          ...
     etc.
```